TripalCultivate/TripalCultivate-Theme
# Setup content region

This PR will setup the content region. No theming applied to the content.

![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/8f044fc9-292e-4d3f-a2da-a38d09e53919)


Note: This PR has a small update to hook preprocess block to adjust the the check to target the tiles content block.

![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/b6dd3de3-c778-477b-81fa-7ba8a6d3132f)

To test:
1. Navigate to my site/admin/structure/block (scroll to the bottom to see new layout region)
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/f42ec2de-e995-429e-a3e6-898cc22bfdc6)
2. Click Place block and place content. It can be any of the predefined content or custom content though my site/admin/content ie: add Basic Block content.

![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/8551bfa2-8a5b-4740-86fc-5af0e7faccea)

Check the Display title configuration to display the title. Otherwise, the title is omitted.

![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/ea13fc88-2022-4aaf-ab97-c82d5c43b0b3)
